api/tasks/http.go

```diff
@@ -105,6 +105,13 @@ func GetTaskOutput(w http.ResponseWriter, r *http.Request) {
 
 func RemoveTask(w http.ResponseWriter, r *http.Request) {
 	task := context.Get(r, "task").(db.Task)
+	editor := context.Get(r, "user").(*db.User)
+
+	if editor.Admin != true {
+		log.Warn(editor.Username + " doesn't permit task log deletion")
```

Comment:
looks like this file is missing a log import making the tests fail currently

Reply:
Oops, my mistake. Fixed for now.